components/feature/accounts-push/src/main/java/mozilla/components/feature/accounts/push/SendTabUseCases.kt

```diff
@@ -174,3 +182,11 @@ internal inline fun filterSendTabDevices(
         }
     }
 }
+
+@VisibleForTesting
+internal fun isValidTabSchema(tab: TabData): Boolean {
+    // We don't sync certain schemas, about|resource|chrome|file|blob|moz-extension
+    // See https://searchfox.org/mozilla-central/source/modules/libpref/init/all.js#4372
+    val filteredSchemas = arrayOf("about:", "resource:", "chrome:", "file:", "blob:", "moz-extension:")
```

Comment:
Does this mean we only want http/https URIs? I'm fine with matching against desktop logic here, but wouldn't it be easier to have a allow list instead of a deny list? We do have things like `tel:` and `mailto:` links as well.

Reply:
I was contemplating doing that too but I think we don't necessarily want to only allow http/https as there could be some valid schemes that would be perfectly compatible cross-platform (mailto: for example). I think the problem really only lay in `file://` since that is showing up as a crasher for people, most likely due to IO. The others in the above list would probably just err/404 if they got through.

Happy to adjust it if we feel strongly but I think the safer choice is to make the feature pair with the desktop equivalent until we can eventually decide on a more cohesive plan across clients (something like -- bring it to the rust layer for all consumers)